lib/airbrake/rack/http_params_filter.rb

```diff
@@ -10,7 +10,7 @@ class HttpParamsFilter
       def call(notice)
         return unless (request = notice.stash[:rack_request])
 
-        notice[:params] = request.params
+        notice[:params].merge!(request.params)
```

Comment:
I would imagine you want it to be reverse merge.
so your filter can mask the request params from sensitive information